status-im/status-react
# [WIP]  [BUG #2029] Properly handle gas details from payment requests

fixes #2029

### Summary:

Properly parse gas details from a payment request (QR code)

### Steps to test:
- Open Status
- Scan a QR code containing gas details
- See those gas details reflected in wallet

status: wip
